### PR TITLE
Dry-container: add information about stubs

### PR DESCRIPTION
**Pull request by davydovanton**

### CODE REVIEW

File: source/gems/dry-container/testing.html.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+require 'dry/container/stub'
+
+# before stub you need to enable stubs for specific container
+container.enable_stubs!
+container.stub(:redis, "Stubbed redis instance")
+
+container[:redis] # => "Stubbed redis instance"
+```
+
+Also, you can unstab container:
```
Comment (AMHOL) [inline source/gems/dry-container/testing.html.md]: typo: `unstab` => `unstub`

Comment (davydovanton) [inline source/gems/dry-container/testing.html.md]: of course, thanks for catch 👍

**Comment by davydovanton** [comment]
@AMHOL done 👍

**Comment by timriley** [comment]
This _could_ be done as a follow-up to this PR, but if we're adding a section about testing, it'd be great to preface it with some general information/guidance about testing practices _in general_ when working with apps based around containers?

For example, if the container is helping you design objects that work with DI, then it'd be helpful to illustrate passing in mock dependencies for unit tests, and perhaps mention that this is preferable to stubbing the container, which should be used in a situation where passing mock dependencies isn't practical?

**Comment by AMHOL** [comment]
Thank you @davydovanton

**Pull request merged**